web/src/test/java/org/springframework/security/web/savedrequest/RequestCacheAwareFilterTests.java

```diff
@@ -51,6 +51,7 @@ public void doFilterWhenCookieRequestCacheConfiguredThenExpiredSavedRequestCooki
 
 		MockHttpServletRequest request = new MockHttpServletRequest();
 		request.setServerName("abc.com");
+		request.setServletPath("/destination");
```

Comment:
We should not have to set the servlet path for this test to pass.

Reply:
I fixed these issues, please check.